en/frames_autogyro/README.md

```diff
@@ -0,0 +1,12 @@
+# Autogyro Frame Builds
+
+[Autogyro](https://en.wikipedia.org/wiki/Autogyro) is a type of [rotor-wing](https://en.wikipedia.org/wiki/Rotorcraft). Compared to other designs it possesses following advantages:
+
+  * Ability to take off and land using only a very short runway.
+  * High resistance to weather conditions, especially gusts of wind.
+  * Possession of a non-powered rotor enabling it to operate in an autorotation mode (one of the helicopter’s emergency modes as well). Consequently, it does not need to actively change a flight mode in case of failure (it does not need a parachute or other actively working devices). Its flight is thus inherently stable at all times.
+  * Absence of a [downwash](https://en.wikipedia.org/wiki/Downwash) during take-off or landing creating an unwanted swirling of dust.
+  * Low lift-to-drag ratio (https://en.wikipedia.org/wiki/Lift-to-drag_ratio) that can be adjusted by construction parameters. This ability can be useful because an unmanned autogyro cannot, in case of failure, fly too far (as is the case of a conventional airplane), but, at the same time, the flight is still safe and aircraft does not fall (as is typical for multicopter or helicopter).
+
+PX4 supports several autogyro airframes. The complete set of supported configurations can be seen in [Airframes Reference > Autogyro](../airframes/airframe_reference.md#autogyro).

```

Comment:
Several? I can only see that one?

Reply:
We would like to publish our airframe of training autogyro [TF-G2](https://github.com/ThunderFly-aerospace/TF-G2/) to upstream within about a month.

Reply:
I understand that. When you push your TF-G2 there will be just one autogyro - the TF-G2. But this says there will be several (more than one).